rootfs/templates/wrapper

```diff
@@ -85,6 +89,7 @@ function use() {
 		if [ -n "${ENV_FILE}" ]; then
 			DOCKER_ARGS+=(--env-file ${ENV_FILE})
 		fi
+		DOCKER_ARGS+=(--env HOST_HOME=$HOME)
```

Comment:
Keep in mind `HOME` is "complicated" since we work with "WSL" as well. That's why we extrapolate the `LOCAL_HOME`